<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UpdateUser</class>
 <widget class="QDialog" name="UpdateUser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1157</width>
    <height>898</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>30</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <widget class="QLabel" name="label_edituser_pwd_2">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>460</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>确认新密码</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lEdit_edituser_user">
   <property name="geometry">
    <rect>
     <x>614</x>
     <y>200</y>
     <width>361</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="lEdit_edituser_disp_name">
   <property name="geometry">
    <rect>
     <x>614</x>
     <y>280</y>
     <width>361</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_edituser_disp_name">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>290</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>显示名</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_edituser_pwd">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>360</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>新密码</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cBox_edituser_type">
   <property name="geometry">
    <rect>
     <x>614</x>
     <y>550</y>
     <width>221</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>审核权限</string>
   </property>
   <property name="iconSize">
    <size>
     <width>36</width>
     <height>36</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pButton_edituser_ok">
   <property name="geometry">
    <rect>
     <x>424</x>
     <y>680</y>
     <width>301</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/images/confirmbt.png);</string>
   </property>
   <property name="text">
    <string>     确认修改</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pButton_edituser_back">
   <property name="geometry">
    <rect>
     <x>794</x>
     <y>680</y>
     <width>301</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/images/back.png);</string>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lEdit_edituser_pwd_2">
   <property name="geometry">
    <rect>
     <x>614</x>
     <y>440</y>
     <width>361</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_edituser_user">
   <property name="geometry">
    <rect>
     <x>474</x>
     <y>220</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>帐号</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lEdit_edituser_pwd">
   <property name="geometry">
    <rect>
     <x>614</x>
     <y>360</y>
     <width>361</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_background">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>150</y>
     <width>711</width>
     <height>471</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/images/loginbk.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label_background</zorder>
  <zorder>label_edituser_pwd_2</zorder>
  <zorder>lEdit_edituser_user</zorder>
  <zorder>lEdit_edituser_disp_name</zorder>
  <zorder>label_edituser_disp_name</zorder>
  <zorder>label_edituser_pwd</zorder>
  <zorder>cBox_edituser_type</zorder>
  <zorder>pButton_edituser_ok</zorder>
  <zorder>pButton_edituser_back</zorder>
  <zorder>lEdit_edituser_pwd_2</zorder>
  <zorder>label_edituser_user</zorder>
  <zorder>lEdit_edituser_pwd</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
